### PR TITLE
Tightening Up Intro (1)

### PR DESCRIPTION
**Pull request by seanturner**
Cleaning up intro. Also adds references to Kyber in CMS and AUTHKEM as well as ASN.1 references.

I almost included that "Kyber is an IND-CCA2-secure key-encapsulation mechanism (KEM)", but figured we could into IND-CCA2 in the security considerations as few are going to have a clue about the significance of IND-CCA2.

NOTE: Also see #26, which is an alternative intro.

### CODE REVIEW

File: draft-ietf-lamps-kyber-certificates.md [hunk 1]
```diff
@@ -41,6 +41,22 @@ author:
...
+      org: ITU-T
+      seriesinfo:
+        ISO/IEC: 8824-1:2021
+  X.690:
+    target: https://www.itu.int/rec/T-REC-X.690
+    title: "Information technology - Abstract Syntax Notation One (ASN.1): ASN.1 encoding rules: Specification of Basic Encoding Rules (BER), Canonical Encoding Rules (CER) and Distinguished Encoding Rules (DER)"
+    date: Feburary 2021
+    author:
+      org: ITU-T
+      seriesinfo:
```
Comment (seanturner) [inline draft-ietf-lamps-kyber-certificates.md:58]:
```suggestion
    seriesinfo:
```

File: draft-ietf-lamps-kyber-certificates.md [hunk 2]
```diff
@@ -41,6 +41,22 @@ author:
     email: bas@westerbaan.name
 
 normative:
+  X.680:
+    target: https://www.itu.int/rec/T-REC-X.680
+    title: "Information technology - Abstract Syntax Notation One (ASN.1): Specification of basic notation"
+    date: Feburary 2021
+    author:
+      org: ITU-T
+      seriesinfo:
```
Comment (seanturner) [inline draft-ietf-lamps-kyber-certificates.md:50]:
```suggestion
    seriesinfo:
```

**Review by seanturner — COMMENTED** [review]
Fixing spacing...grrr

**Review by seanturner — COMMENTED** [review]
Fix refs.

File: draft-ietf-lamps-kyber-certificates.md [hunk 3]
```diff
@@ -74,24 +89,24 @@ and will use placeholders until these are released.]
...
-is used to encapsulate a shared secret used to derive a symmetric key
-used to encrypt content in CMS
-\[EDNOTE: Add reference draft-perret-prat-lamps-cms-pq-kem\].
-To be used in TLS, these certificates could only be used as end-entity
-identity certificates and would require significant updates to the
-protocol
-\[EDNOTE: Add reference draft-celi-wiggers-tls-authkem\].
+Kyber is a key-encapasulation mechanism (KEM). In 2022, the US NIST PQC
+Project {{PQCProj}} selected Kyber as the public-key encryption and
+key-establishment winning algorithm. NIST has also defined object
```
Comment (csosto-pk) [inline draft-ietf-lamps-kyber-certificates.md:94]: Remove "winning". "selected Kyber" sounds better.

**Review by csosto-pk — COMMENTED** [review]
Changes look fine assuming the minor comments are addressed. 

I slightly like this one more, but I could live with the other intro as well.

**Comment by seanturner** [comment]
Seem to be leaning towards #26 after mods from @csosto-pk.

**Comment by seanturner** [comment]
Closed in favor of #26.

**Pull request closed without merging**